test/functional/commands/url-e2e-specs.js

```diff
@@ -1,29 +1,54 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import AndroidDriver from '../../..';
+import ADB from 'appium-adb';
+
 
 chai.should();
 chai.use(chaiAsPromised);
 
 let driver;
 let defaultCaps = {
-  appPackage: 'com.android.browser',
-  appActivity: '.BrowserActivity',
+  browserName: 'Browser',
   deviceName: 'Android',
   platformName: 'Android'
 };
 
-describe('setUrl', function (){
-  before(async () => {
+describe('setUrl', function () {
+  let urlId = 'com.android.browser:id/url';
+  before(async function () {
+    let adb = new ADB();
+    if (!await adb.isAppInstalled('com.android.browser')) {
+      if (!await adb.isAppInstalled('com.android.chrome')) {
+        throw new Error('Neither default browser nor chrome available');
+      }
+      // `browser` is not available, so use `Chrome`
+      defaultCaps.browserName = 'Chrome';
+      urlId = 'com.android.chrome:id/url_bar';
+    }
+
     driver = new AndroidDriver();
     await driver.createSession(defaultCaps);
   });
   after(async () => {
     await driver.deleteSession();
   });
+
   it('should be able to start a data uri via setUrl', async () => {
+    if (defaultCaps.browserName >= 'Chrome') {
```

Comment:
Yes, that was a typo (I did all this on the plane, and it must have slipped through).